### DISCUSSION TITLE
Allow grief prevention without claims

### DISCUSSION THREAD

**Discussion by hades200082** [GriefPrevention/GriefPrevention#272] [Q&A] [answered]
I'd like to be able to have the grief prevention settings like preventing creeper explosions above sea level and the chat & admin tools without having to have claims enabled.

I use Factions for claims.

Is this possible?

**Comment by bigpresh** [comment]
You can disable the ability to create claims - see the permissions doco - and if you set the world type to Survival, then claims aren't required to build or anything.

**Comment by RoboMWM** [comment, marked as answer]
He wants the "wilderness" protections tho. I think I requested this a long time ago.

The current solution is to set as survival, but disable claiming via permissions.
